plugins/harness-ci-cd/src/components/ExecutionList/ExecutionList.tsx

```diff
@@ -956,6 +956,7 @@ function ExecutionList() {
       });
     }
     if (toggle) {
+      // eslint-disable-next-line @typescript-eslint/no-shadow
       const response = await fetch(
```

Comment:
you can rename `response` to maybe `res` or `apiRes` to resolve this eslint issue.

Reply:
Done, removed this disable